docs/react/use-with-next.en-US.md

```diff
@@ -33,8 +33,9 @@ Now we install `antd` from yarn or npm or pnpm.
 Modify `src/app/page.tsx`, import Button component from `antd`.
 
 ```tsx
-'use client'; // If used in Pages Router, is no need to add "use client"
+'use client';
```

Comment:
The format here should not change

Reply:
So that's it, you can manually edit on GitHub to revert this line of code

Reply:
Done, thanks for the tip 🙏

Reply:
You're welcome, and I need to verify if the same problem exists on the Pages Router.

Reply:
I did that and I didn't notice the issue on the Pages Router